bin/generate_structs.jl

```diff
@@ -8,8 +8,20 @@ template = """
 #=
 This file is auto-generated. Do not edit.
 =#
+\"\"\"
```

Comment:
Would you mind deleting this file?  It should not be needed any longer and causes confusion.

Reply:
@jd-lara the auto-generation happens in InfrastructureSystems src/utils/generate_structs, so there is no need for the bin/generate_Structs it's deleted now